src/components/WorkOrders/__snapshots__/JobRow.test.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`JobRow component should render properly 1`] = `
+<DocumentFragment>
+  000123452021-01-12T16:24:26.632
```

Comment:
I think we need to update the component test for this snapshot to pass in Date objects rather than strings for `lastUpdated` and `dateRaised`. At the moment this part of the snapshot just gives us the string we passed in for those values rather than our human-friendly formatted dates / times.